doc/releases/changelog-dev.md

```diff
@@ -160,6 +160,9 @@
   its functionality from the `QuantumTape` class.
   [(#3401)](https://github.com/PennyLaneAI/pennylane/pull/3401)
 
+* Replace `qml.transforms.make_tape` with the new `qml.tape.make_qscript` across the codebase.
```

Comment:
Maybe:
```suggestion
* Adds `qml.tape.make_qscript` for converting a quantum function into a quantum script.
  Replaces `qml.transforms.make_tape` with `make_qscript`.
```

Don't want to bury the addition of the `make_qscript` function.

Reply:
I tried to imply the addition with "new". Also I realize that the docstring says this as well, but if I remember correctly, we define a quantum function as a function that returns an observable. However, make_qscript works for any sequence of Operators etc without complaint. It's probably not bad to support more cases than the docs suggest, just confirming the distinction